tests/roles/ceph_migrate/defaults/main.yml

```diff
@@ -30,10 +30,15 @@ ceph_timeout: 30
 # updated
 ceph_wait_mon_timeout: 10
 ceph_keep_mon_ipaddr: true
-ceph_firewall: false
-firewall_path:
+
+# firewall section
+ceph_firewall_enabled: false
+ceph_iptables_path:
   - "/etc/sysconfig/iptables"
   - "/etc/sysconfig/ip6tables"
+ceph_nftables_path:
+  - "/etc/nftables/tripleo-rules.nft"
```

Comment:
The current jobs might still have 17.1.3 content, with `iptables` instead of `nftables`, so we probably need to hold this for a bit.

Reply:
Adopted computes already have `nftables`.